end-to-end/ratelimit-service/ratelimit.proto

```diff
@@ -0,0 +1,99 @@
+// FROM https://github.com/envoyproxy/envoy/blob/master/source/common/ratelimit/ratelimit.proto
```

Comment:
Should we just copy this proto in from the envoyproxy github? Is your intention here to pin to specific proto version?